### PR TITLE
allow outputMappings for all events

### LINKED ISSUE #242: I can define output variable mappings on all events
__Is your feature request related to a problem? Please describe.__

Currently, I can define output variable mappings only for message start/intermediate/boundary events. For some use cases, it would be useful to define the mappings also for other types of events. For example:
* none start event: initialize variables that are used later in the workflow
* none end event: collect/aggregate variables that are sent back as response of the create instance command 

Zeebe is supporting output mappings for all events already.

__Describe the solution you'd like__

I can define output variable mappings on all kinds of start, intermediate, boundary, and end events.

__Describe alternatives you've considered__

If I can not define the output mappings in the workflow then the transformation/creation of the variables needs to be done in job worker or application.

__Additional context__

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Closes #242

__Acceptance Criteria__

* [x] Corresponds to the concept: _I can define output variable mappings on all kinds of start, intermediate, boundary, and end events._
* [x] Corresponds to the design: _uses existing constructs / design_

__Definition of Done__

* [x] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [x] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [x] available as feature branch on GitHub
  * [x] contains the cleaned up commit history
  * [x] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [x] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js [hunk 1]
```diff
@@ -125,14 +123,17 @@ export function areInputParametersSupported(element) {
...
     'bpmn:SubProcess',
     'bpmn:ReceiveTask',
-    'bpmn:CallActivity'
-  ]) || messageEventDefinition;
+    'bpmn:CallActivity',
+    'bpmn:StartEvent',
+    'bpmn:EndEvent',
+    'bpmn:IntermediateThrowEvent',
+    'bpmn:IntermediateCatchEvent',
+    'bpmn:BoundaryEvent'
```
Comment (barmac) [inline client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js:135]: How about:
```suggestion
    'bpmn:Event'
```
instead?

Comment (MaxTru) [reply client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js]: :exploding_head:

Comment (barmac) [reply client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js]: That would reflect this requirement explicitly:

> Zeebe is supporting output mappings for all events already.

Comment (barmac) [reply client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js]: I missed your response, sorry 😅

**Pull request merged**